examples/http/setup.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const { NodeTracer } = require('@opentelemetry/node-sdk');
+const { SimpleSpanProcessor } = require('@opentelemetry/basic-tracer');
+const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
+const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+const EXPORTER = process.env.EXPORTER || '';
+
+function setupTracerAndExporters(service) {
+    let exporter;
+    const options = {
+        serviceName: service,
+    }
+    const tracer = new NodeTracer({
+        plugins: {
+            http: {
+                enabled: true,
+                // if it can't find the module, put the absolute path (depending your config, since packages are not published yet)
+                path: '@opentelemetry/plugin-http',
+                ignoreOutgoingUrls: [/spans/]
+            }
+        }
+    });
+    if (EXPORTER.toLowerCase().startsWith('z')) {
+      // need ignoreOutgoingUrls: [/spans/] to avoid infinity loops
+      // TODO: manage this situation
+      const zipkinExporter = new ZipkinExporter(options);
+      exporter = new SimpleSpanProcessor(zipkinExporter);
```

Comment:
`s/exporter/processor/g` ?

Reply:
Also, I think we should use `BatchSpanProcessor` in case of Zipkin exporter (dependency on #238). Jaeger library has its own batching mechanism, so it makes sense to use `SimpleSpanProcessor` with Jaeger exporter.